spec/support/fake_heroku.rb

```diff
@@ -6,8 +6,9 @@ def initialize(args)
   end
 
   def run!
-    if @args.first == "plugins"
-      puts "heroku-pipelines@0.29.0"
+    if @args.first == "help"
+      # spaces in front are intentional
+      puts "  pipelines      #  manage collections of apps in pipelines"
```

Comment:
I understand that the spaces are required because of the regex, but are they mandatory? Can't we change the regex?